toolchain/sem_ir/ids.cpp

```diff
@@ -15,8 +15,8 @@ auto InstId::Print(llvm::raw_ostream& out) const -> void {
     return;
   }
   out << Label;
-  if (is_builtin()) {
-    out << "(" << builtin_inst_kind() << ")";
+  if (index < static_cast<int>(SingletonInstKinds.size())) {
```

Comment:
Can you use `IsSingletonInstId` here?

Reply:
I could, but I thought this might be a little clearer with the subtraction a few lines below. Thoughts?

Reply:
Yeah, I see your point, and I don't really feel strongly either way.

For what it's worth, I've found the subtraction to be unhelpful, because it makes it hard to figure out how dumped `InstId`s correspond to indexes that I see while debugging, so I'd be happy to get rid of that subtraction at some point. It saves us from churn when adding more singleton instructions, but I think ideally we should be aiming for a place where `InstId`s formatted this way don't show up in formatted `SemIR` at all, which should minimize the churn. Maybe we should leave this as a delta until we get `import_ref` printing to not print the raw `InstId` though, since currently we do print a lot of those in our formatted SemIR.